### PR TITLE
Fix name

### PR DESCRIPTION
**Pull request by remyLemeunier**

### CODE REVIEW

File: context/context.go [hunk 1]
```diff
@@ -11,14 +11,15 @@ import (
...
 
 type Context struct {
-	Deployer        deployers.Deployer
-	Vcs             services.Sources
-	Binaries        services.Binaries
-	Hooks           []hooks.Hooks
-	LockSystem      utils.Lock
-	ScreenMandatory bool
-	Log             *log.Logger
-	Metrics         *utils.MetricsRegistry
```
Comment (obiesmans) [inline context/context.go]: :/

Comment (remyLemeunier) [reply context/context.go]: ?

File: README.md [hunk 2]
```diff
@@ -94,6 +94,9 @@ lockSystem:                # Define a lock system in order to avoid multiple com
   fileLock:                # Currently we only have the lock by file.
     filePath: /tmp         # Path where the is going to be written.
 
+potentialUsername:       # By default get the current user.
+  - USER                 # You can also have a list of env variable
+  - bamboo.jira.username # where you can find the username
```
Comment (obiesmans) [inline README.md]: Maybe add that the fields are tested in order ?

File: context/context.go (hunk 1 above)
Comment (obiesmans) [reply context/context.go]: A bad format that still passed through our tests ?

File: README.md (hunk 2 above)
Comment (remyLemeunier) [reply README.md]: Good point :)

**Pull request merged**